### DISCUSSION TITLE
How to excecute dragon-drop

### DISCUSSION THREAD

**Discussion by CRAG666** [sxyazi/yazi#327] [1. Q&A] [answered]
I want to use Dragon-Drop, to use Drag and Drop in selected files or the currently selected, how can I create a keymap for this action

**Comment by sxyazi** [comment, marked as answer]
Try like this:

```bash
{ on = "<C-n>", run = "shell -- dragon -x -i -T %s1" }
```

https://github.com/mwh/dragon

**Reply by SolitudeSF** [reply]
that doesnt do anything on my machine.

**Reply by sxyazi** [reply]
ah you're right, it should be:

```bash
{ on = "<C-n>", run = "shell -- dragon -x -i -T %s1" }
```


https://github.com/sxyazi/yazi/assets/17523360/3ffed004-714c-4d9c-a741-ff56a8dd7482

**Comment by knodalyte** [comment]


**Reply by NippleOfAnApe** [reply]
Some package manager change an executable name. For example on Void Linux the name is not `dragon` but `dragon-drop`. `nnn` has a plugin which finds  a name like this [dragdrop](https://github.com/jarun/nnn/blob/f2a8648861ff88e1c2d5a95ae4d76b7367e12c9b/plugins/dragdrop#L20)

Also you probably want to pass `"$@"` instead of `"$1"` so that every file you selected will be dropped

**Comment by Vaisakhkm2625** [comment]
for [ripdrag](https://github.com/nik012003/ripdrag)


```sh
{ on = "<C-n>", run = "shell -- ripdrag %s -x 2>/dev/null &" }
```
